### PR TITLE
SlopeClock++ README update and debug feature for PrimeTimeLato

### PR DESCRIPTION
**Pull request by hughbarney**

### CODE REVIEW

File: apps/slopeclockpp/README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ to change (top right or  bottom left). It should change color showing
 it is selected.
 
 * Swipe up or down to cycle through the info screens that can be displayed
+  when you have finnised tap again towards the centre of the screen to unselect.
```
Comment (myxor) [inline apps/slopeclockpp/README.md:13]: typo. should be "finished".

Comment (gfwilliams) [reply apps/slopeclockpp/README.md]: thanks - just tweaked

**Comment by gfwilliams** [comment]
Thanks!

**Comment by gfwilliams** [comment]
Just to say, if you update the README for things, don't worry about bumping the version/changelog - as there won't be any point in the user updating the app itself

**Comment by hughbarney** [comment]
Thanks Gordon, makes sense.  Will do that in future.

**Pull request merged**